web/README.md

```diff
@@ -85,39 +89,32 @@ Next.js can be deployed to any hosting provider that supports Node.js. Make sure
 }
 ```
 
-`next build` builds the production application in the `.next` folder. After building, `next start` starts a Node.js server that supports hybrid pages, serving both statically generated and server-side rendered pages.
+`yarn build` builds the production application in the `.next` folder. After building, `yarn start` starts a Node.js server that supports hybrid pages, serving both statically generated and server-side rendered pages.
```

Comment:
@ch3yne would you please review this?

Reply:
The following line introduces the built-in methods used in the scripts, so there is no need to change it to `yarn`.

Reply:
@justinmclean would you please revert this line's change?

Reply:
The original document was incorrect. It was used next in the description but not in the code locks. This would be confusing to a user.

Reply:
We should be consistent and either use next or yarn and not mix them.

Reply:
The result remains consistent after runing `next build` or `yarn build`.
Based on the semantics, I acknowledge that it can be changed to `yarn`.